Running-Mastodon/PgBouncer-guide.md

```diff
@@ -13,13 +13,18 @@ Note that you can check "PgHero" in the administration view to see how many Post
 Installing PgBouncer
 -----
 
-On Ubuntu:
+On Ubuntu/Debian:
 
     sudo apt install pgbouncer
 
-Restarting:
+Then:
 
-    sudo service pgbouncer restart
+    nano /etc/default/pgbouncer
+    START=1
```

Comment:
When I was trying to understand why it didn't start for me. I ended up [here](http://stackoverflow.com/questions/20249385/how-to-get-pgbouncer-auto-start-on-reboot-on-linux)
I had tried `service pgbouncer restart` and the service didn't start. When I found that and made the change to `START=1` it started!? 
Also, now that I look at that link, shouldn't we also include `/etc/init.d/pgbouncer start` in the guide?